flytekit/models/schedule.py

```diff
@@ -158,7 +158,7 @@ def from_flyte_idl(cls, pb2_object):
         :param flyteidl.admin.schedule_pb2.Schedule pb2_object:
         :rtype: Schedule
         """
-        return cls(
+        return Schedule(
```

Comment:
This is cuz you explicitly don't want the inheritance?  Add a comment if that's the case?